### DISCUSSION TITLE
App dependencies

### DISCUSSION THREAD

**Discussion by b0ndemand** [Micke-K/IntuneManagement#159] [Ideas]
Hi, thanks for the great tool. We use it a lot!

I was wondering if it could be possible to also export app dependencies when exporting an application. So that it can grab both applications (of course you would still need the intunewin file on the folder).

**Comment by Micke-K** [comment]
Hello!
I've started looking at this so I hope to get it in at next release. 
Cheers!

**Comment by Micke-K** [comment]
Hello!

Can you try with the attached files?

Apps will be imported without dependency/supersedence. These will be added after all apps are imported.

Bulk Import should take care of all references.

For single file import, any referenced apps must exist to the import. 

You must export new json files for apps before this will work. Additional information is added to the json files.

The script will look for reference apps based on name and version. Name is case sensitive so it must match.

All reference association will be logged so check the log file if it is not working.

Let me know how it goes.

Cheers!

[MSGraph.zip](https://github.com/Micke-K/IntuneManagement/files/12836960/MSGraph.zip)

**Comment by b0ndemand** [comment]
Hi Mikael, so sorry for not getting back to you. You are so kind to provide quick updates!
But i can see you have added it to the new version. We will start using that and report back if we have any issues :-)
